ironic/drivers/modules/drac/raid.py

```diff
@@ -202,20 +253,77 @@ def delete_virtual_disk(node, virtual_disk):
         raise exception.DracOperationError(error=exc)
 
 
-def commit_config(node, raid_controller, reboot=False):
+def reset_config(node, raid_controller):
+    """Delete all virtual disk and unassign all hotspares physical disk
+
+    :param node: an ironic node object.
+    :param raid_controller: id of the RAID controller.
+    :returns: a dictionary containing the is_commit_required needed key with a
+              boolean value indicating whether a config job must be created
+              for the values to be applied.
+    :raises: DracOperationError on an error from python-dracclient.
+    """
+    drac_job.validate_job_queue(node)
+
+    client = drac_common.get_drac_client(node)
+
+    try:
+        return client.reset_config(raid_controller)
+    except drac_exceptions.BaseClientException as exc:
+        LOG.error('DRAC driver failed to delete all virtual disk '
+                  'and unassign all hotspares '
+                  'for node %(node_uuid)s. '
+                  'Reason: %(error)s.',
+                  {'raid_controller_fqdd': raid_controller,
+                   'node_uuid': node.uuid,
+                   'error': exc})
+        raise exception.DracOperationError(error=exc)
+
+
+def clear_foreign_config(node, raid_controller):
+    """Free up the foreign drivers.
```

Comment:
Can you change "drivers" to "drives" in the line above?

Reply:
already changed..It will reflect into new commit

Reply:
Done.